### ISSUE TITLE
Terraform Provider Error with unsupported Attribute

### ISSUE THREAD

**Issue by theurichde** [aiven/terraform-provider-aiven#748]
Hi folks,

we are currently trying to upgrade the terraform provider from version 2.7.3 to 3.0.0

When I try to import a resource  "aiven_vpc_peering_connection.peering_connection" it fails with the provider saying "_Invalid resource instance data in state_".

I also tried to use / import the new un-deprecated aiven_aws_vpc_peering_connection which fails too, with the same error.

What I can see in my terraform state, is, that the aiven_project resource has an empty billing address set.

Is there some kind of migration guide from 2.7.x to 3.0.0? We didn't see anything like that on the README, release notes or docs.

```
$ terragrunt import 'module.peering_connections.aiven_vpc_peering_connection.peering_connection["31884025XXXX/vpc-05717a4a7aaXXXXXX"]' cloud-shuttle-stg-idealo/8c5b9cd7-7ea9-45bf-a137-6c831f0234eb/31884025XXXX/vpc-05717a4a7aaXXXXXX
[INFO] Getting version from tgenv-version-name
[INFO] TGENV_VERSION is 0.37.1
module.peering_connections.aiven_vpc_peering_connection.peering_connection["31884025XXXX/vpc-05717a4a7aaXXXXXX"]: Importing from ID "cloud-shuttle-stg-idealo/8c5b9cd7-7ea9-45bf-a137-XXXXXX/31884025XXXX/vpc-05717a4a7aaXXXXXX"...
module.peering_connections.aiven_vpc_peering_connection.peering_connection["31884025XXXX/vpc-05717a4a7aaXXXXXX"]: Import prepared!
  Prepared aiven_vpc_peering_connection for import
module.peering_connections.aiven_vpc_peering_connection.peering_connection["31884025XXXX/vpc-05717a4a7aaXXXXXX"]: Refreshing state... [id=cloud-shuttle-stg-idealo/8c5b9cd7-7ea9-45bf-a137-6c831f0234eb/31884025XXXX/vpc-05717a4a7aaXXXXXX]
╷
│ Error: Invalid resource instance data in state
│ 
│   on /home/tim.heurich/projects/CST/aiven-projects/idealo/projects/cloud-shuttle/stg/.terragrunt-cache/9JMIr47_m5qP8s0Gx9CwgRupax4/87fpqfXsLuHdNfcL8z1vaiuyqpY/modules/aiven-project/main.tf line 1:
│    1: resource "aiven_project" "aiven_project" {
│ 
│ Instance aiven_project.aiven_project data could not be decoded from the
│ state: unsupported attribute "billing_address".
╵

ERRO[0007] 1 error occurred:
        * exit status 1
          

```

**Comment by wanisfahmyDE** [comment]
Hi @ivan-savciuc , do you have an idea here what might be wrong?
we're currently unable at the moment to upgrade to V3.0.0
Thank you

**Comment by ivan-savciuc** [comment]
Hi @theurichde and @wanisfahmyDE, project resource has gone through some changes. All the billing related field were deprecated in last iteration of v2 and in v3 we deleted them completely.

The idea is that all billing related fields should be set in a `aiven_billing_group` resource, and `aiven_project` resource has a `billing_group_id` field that can be ussed to associate a billing group to a project. 

So in our case, if a project has no billing group we should create one and set all the billing related fields from the project resource to the coresponding billing group resource. 

We do not have v2 -> v3 migration guide, and it looks like we need to create one, we will add it shortly. 

Please let me know if the billing group resource solves this issue.

**Comment by dangquangdon** [comment]
Maybe it worth mentioning that `billing_address` is deprecated in favor of `address_lines`

https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/billing_group

**Comment by theurichde** [comment]
Hi @ivan-savciuc and @dangquangdon 

We already use the billing_group with the billing_group.id - that was not the problem. 

When I imported a resource doing a `terraform apply` with the updated provider, it crashed due to the presence of the billing_address attribute in the state file. 

I fixed the issue with an `apply` before importing the resource. So the empty billing_address attribute got deleted from state. 

The import works now.
